src/ngraph/frontend/onnxifi/onnxifi.cpp

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <cstdint>
+#include <onnxifi.h>
+
+extern "C" {
+
+onnxStatus onnxGetBackendIDs(onnxBackendID* backendIDs, std::size_t* numBackends)
```

Comment:
Function names are snake_case

Reply:
@rkimballn1 We are implementing the ONNXIFI interface, which forces us to use these camelCase names. All names listed here:
https://github.com/onnx/onnx/blob/master/onnx/onnxifi.h#L932

Reply:
@apwojcik Shouldn't we be appending `NGRAPH` to all the function names, so this should be `onnxGetBackendIDsNGRAPH`?

Reply:
No, the documentation is not up to date. Please take a look at loader implementation
https://github.com/onnx/onnx/blob/master/onnx/onnxifi_loader.c